src/apps/watchfaces/OswAppWatchfaceMonotimer.cpp

```diff
@@ -0,0 +1,143 @@
+
+#include "./apps/watchfaces/OswAppWatchfaceMonotimer.h"
+// #define GIF_BG
+
+#ifdef ANIMATION
+#include <anim_matrix.h>
+#endif
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+
+#ifdef GIF_BG
+#include "./apps/_experiments/gif_player.h"
+#endif
+
+void OswAppWatchfaceMonotimer::drawNShiftedTicks(Graphics2D *gfx, uint8_t cx, uint8_t cy, uint8_t r1, uint8_t r2, uint8_t nTicks, float shift, uint16_t color) {
+    const float deltaAngle = 360.0 / nTicks;
+    for(uint16_t i=0; i<nTicks; ++i)
```

Comment:
All if, for statements have `{`, `}` Please use parentheses

Reply:
Ok, I fixed it.